src/views/PlanningOverview/PlanningGrid.tsx

```diff
@@ -35,7 +36,7 @@ export const PlanningGrid = ({ startDate, endDate }: PlanningGridProps): JSX.Ele
   }, [startTime, endTime, indexUrl, locale])
 
   const { data } = useSWR(searchUrl.href, async (): Promise<PlanningsByDate | undefined> => {
-    if (!jwt) {
+    if (status !== 'authenticated' || !session?.accessToken) {
```

Comment:
We will do this a lot. Even though it is little code this could be a strong candidate for a function call like `!isValidSession(session) { return }`. Would make it readable, easier to extend/change and...

Reply:
> We will do this a lot. Even though it is little code this could be a strong candidate for a function call like `!isValidSession(session) { return }`. Would make it readable, easier to extend/change and...

Yes, the reason here was mostly because of types, I think `session` is `Session | undefined` but if we are `authenticated` we should also have an `accessToken`, so it feels a bit superfluous. I need to look more closer to this.